amplify-tools-gradle-plugin/build.gradle

```diff
@@ -0,0 +1,25 @@
+apply plugin: 'groovy'
+apply plugin: 'idea'
+apply plugin: 'maven-publish'
+
+apply plugin: 'com.android.library'
+
+sourceSets.main.groovy.srcDirs = ["src/main/java", "src/main/groovy"]
+
+dependencies {
+    implementation 'androidx.appcompat:appcompat:1.1.0'
+}
+
+publishing {
```

Comment:
This is integrated into the root `build.gradle`, so doesn't need to be present in this module-level gradle.